src/planner/parabolic/HauserParabolicSmootherHelpers.cpp

```diff
@@ -13,172 +13,184 @@ namespace planner {
 namespace parabolic {
 namespace detail {
 
-class SmootherFeasibilityCheckerBase : public ParabolicRamp::FeasibilityCheckerBase
+class SmootherFeasibilityCheckerBase
+    : public ParabolicRamp::FeasibilityCheckerBase
 {
 public:
-    SmootherFeasibilityCheckerBase(aikido::constraint::TestablePtr testable,
-                                   double checkResolution)
-        : testable_(std::move(testable)),          
-          checkResolution_(checkResolution),
-          statespace_(testable_->getStateSpace()),
-          interpolator_(statespace_)
+  SmootherFeasibilityCheckerBase(
+      aikido::constraint::TestablePtr testable, double checkResolution)
+    : testable_(std::move(testable))
+    , checkResolution_(checkResolution)
+    , statespace_(testable_->getStateSpace())
+    , interpolator_(statespace_)
+  {
+  }
```

Comment:
Nit: Please add `// Do nothing` comment to indicate that this function is intentionally empty (but initializing the member variables).